cms/envs/common.py

```diff
@@ -445,6 +445,7 @@
     # This is how you would use the textbook images locally
     # ("book", ENV_ROOT / "book_images"),
 ]
+DEFAULT_COURSE_ABOUT_IMAGE_URL = 'images/pencils.jpg'
```

Comment:
Instead of duplicating this line here and in lms/envs/common.py, add `DEFAULT_COURSE_ABOUT_IMAGE_URL` to the `from lms.envs.common import` statement at the beginning of this line.